doc/source/reference/apis/v2-protocol.md

```diff
@@ -1,18 +1,18 @@
-# V2 Inference Protocol
+# Open Inference Protocol
 
-The V2 Inference Protocol is an industry-wide effort to provide an standardised
+The Open Inference Protocol is an industry-wide effort to provide a standardized
 protocol to communicate with different inference servers (e.g. MLServer,
 Triton, etc.) and orchestrating frameworks (e.g. Seldon Core, KServe, etc.).
-The spec of the V2 Inference Protocol defines both the endpoints and payload
+The spec of the Open Inference Protocol defines both the endpoints and payload
 schemas for REST and gRPC interfaces.
 
-As part of the V2 Protocol definition, you can find dedicated endpoints for:
+As part of the Open Inference Protocol definition, you can find dedicated endpoints to:
 
-- Health endpoints, to assess liveness and readiness of your model.
-- Inference endpoints, to interact with your model.
-- Metadata endpoints, to query your model metadata (e.g. expected inputs, expected
+- Assess liveness and readiness of your model.
+- Interact with your model.
+- Query your model metadata (e.g. expected inputs, expected
```

Comment:
**thought:** I think these changes make them more vague as to what the endpoints are about. I'd keep the "sections" (Health, Inference, and Metadata).

KServe uses them as "sections" too, e.g. https://kserve.github.io/website/0.10/modelserving/data_plane/v2_protocol/#inference.

Reply:
Yea it was more of a grammar thing that bothered me initially, but have updated to add categories back in

Reply:
I thought it may have been the grammar! That comma was a bit ugly. Thank you for adding the sections back. My last comment on it would be that for an inference protocol, we now don't mention inference at all in the partial list of endpoints 😅